src/main/java/com/robertx22/mine_and_slash/aoe_data/database/exile_effects/adders/ModEffects.java

```diff
@@ -141,14 +135,13 @@ private static EffectCtx state(String id, String name, Elements elem) {
 
     static {
         for (var rt : RES_NAME.keySet()) {
-            var elem = RES_ELEM.get(rt);
             var nice = RES_NAME.get(rt);
 
             LEECHING_STATE_BY_RES.put(
-                rt, state("leeching_" + rt.id + "_state", "Leeching " + nice, elem)
+                rt, state("leeching_" + rt.id + "_state", "Leeching " + nice, null)
```

Comment:
It's always a good behavior to get rid of arguments that can be `null`
```suggestion
                rt, statePhysical("leeching_" + rt.id + "_state", "Leeching " + nice)
```

Reply:
Yeah, same thing here. I can't remove null. It causes errors because of how EffectCtx is setup.